samcli/lib/package/utils.py

```diff
@@ -33,14 +34,49 @@ def make_abs_path(directory, path):
     return path
 
 
-def is_s3_url(url):
+def is_s3_protocol_url(url):
+    """
+    Check whether url is a valid path in the form of "s3://..."
+    """
     try:
         S3Uploader.parse_s3_url(url)
         return True
     except ValueError:
         return False
 
 
+# https://docs.aws.amazon.com/AmazonS3/latest/dev-retired/UsingBucket.html
+_REGION_PATTERN = r"[a-zA-Z0-9-]+"
```

Comment:
Lets move these to the top of file, if they are constants, after all the imports.

Reply:
updated